cardano-crypto-tests/src/Test/Crypto/Vector/Vectors.hs

```diff
@@ -0,0 +1,140 @@
+  map
+    encodeUtf8
+    [ "0",
+      "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE"
+    ]
+
+wrongEcdsaVerKeyTestVector :: VerKeyDSIGN EcdsaSecp256k1DSIGN
+wrongEcdsaVerKeyTestVector = vKeyParser "02D69C3509BB99E412E68B0FE8544E72837DFA30746D8BE2AA65975F29D22DC7B9"
+
+wrongSchnorrVerKeyTestVector :: VerKeyDSIGN SchnorrSecp256k1DSIGN
+wrongSchnorrVerKeyTestVector = vKeyParser "D69C3509BB99E412E68B0FE8544E72837DFA30746D8BE2AA65975F29D22DC7B9"
+
+-- Raw string verification key that is not on the curve should result in ver key parse failed
+verKeyNotOnCurveTestVectorRaw :: HexStringInCBOR
+verKeyNotOnCurveTestVectorRaw = "02EEFDEA4CDB677750A420FEE807EACF21EB9898AE79B9768766E4FAA04A2D4A34"
+
+-- Parse these vectors and expect the errors on tests with parity bit
+wrongLengthVerKeyTestVectorsRaw :: [HexStringInCBOR]
+wrongLengthVerKeyTestVectorsRaw =  
+    [ -- Ver key of length 30 bytes
+      "02DFF1D77F2A671C5F36183726DB2341BE58FEAE1DA2DECED843240F7B50",
+      -- Ver key of length 34 bytes
+      "02DFF1D77F2A671C5F36183726DB2341BE58FEAE1DA2DECED843240F7B502BA659FF"
+    ]
+
+-- Raw hexstring to be used in invalid length signature parser tests for ecdsa
+ecdsaWrongLengthSigTestVectorsRaw :: [HexStringInCBOR]
+ecdsaWrongLengthSigTestVectorsRaw =
+    [ "354b868c757ef0b796003f7c23dd754d2d1726629145be2c7b7794a25fec80a06254f0915935f33b91bceb16d46ff2814f659e9b6791a4a21ff8764b78d7e1",
+      "354b868c757ef0b796003f7c23dd754d2d1726629145be2c7b7794a25fec80a06254f0915935f33b91bceb16d46ff2814f659e9b6791a4a21ff8764b78d7e114FF"
+    ]
+
+-- Raw hexstring to be used in invalid length signature parser tests for schnorr
+schnorrWrongLengthSigTestVectorsRaw :: [HexStringInCBOR]
+schnorrWrongLengthSigTestVectorsRaw =
+    [ "5a56da88e6fd8419181dec4d3dd6997bab953d2fc71ab65e23cfc9e7e3d1a310613454a60f6703819a39fdac2a410a094442afd1fc083354443e8d8bb4461a",
+      "5a56da88e6fd8419181dec4d3dd6997bab953d2fc71ab65e23cfc9e7e3d1a310613454a60f6703819a39fdac2a410a094442afd1fc083354443e8d8bb4461a9bFF"
+    ]
+
+ecdsaMismatchMessageAndSignature :: [(ByteString, VerKeyDSIGN EcdsaSecp256k1DSIGN, SigDSIGN EcdsaSecp256k1DSIGN)]
+ecdsaMismatchMessageAndSignature =
+  map
+    (\(vm, vKey, sig) -> (encodeUtf8 vm, vKeyParser vKey, sigParser sig))
+    --  verifyMessage, vKey, signature
+    [ ("243F6A8885A308D313198A2E03707344A4093822299F31D0082EFA98EC4E6C89", "0325d1dff95105f5253c4022f628a996ad3a0d95fbf21d468a1b33f8c160d8f517", "3dccc57be49991e95b112954217e8b4fe884d4d26843dfec794feb370981407b79151d1e5af85aba21721876896957adb2b35bcbb84986dcf82daa520a87a9f9"), -- wrong verify message but right signature
+      ("FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF", "0325d1dff95105f5253c4022f628a996ad3a0d95fbf21d468a1b33f8c160d8f517", "5ef63d477c5d1572550016ccf72a2310c7368beeb843c85b1b5697290872222a09e7519702cb2c9a65bbce92d273080a0193b77588bc2eac6dbcbfc15c6dfefd") -- right verify message but wrong signature
+    ]
```

Comment:
Consider making those into multiline tuples. I can't even see the end of the line in the github diff while reviewing it